tests/basic_one_way_send.go

```diff
@@ -0,0 +1,175 @@
+package tests
+
+import (
+	"context"
+	"math/big"
+	"time"
+
+	avalancheWarp "github.com/ava-labs/avalanchego/vms/platformvm/warp"
+	warpBackend "github.com/ava-labs/subnet-evm/warp"
+	. "github.com/ava-labs/teleporter/tests/utils"
+
+	"github.com/ava-labs/awm-relayer/messages/teleporter"
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/subnet-evm/interfaces"
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ava-labs/subnet-evm/x/warp"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	teleporterMessage = teleporter.TeleporterMessage{
+		MessageID:               big.NewInt(1),
+		SenderAddress:           FundedAddress,
+		DestinationAddress:      FundedAddress,
+		RequiredGasLimit:        big.NewInt(1),
+		AllowedRelayerAddresses: []common.Address{},
+		Receipts:                []teleporter.TeleporterMessageReceipt{},
+		Message:                 []byte{1, 2, 3, 4},
+	}
```

Comment:
This `teleporterMessage` is only used in one location, and only for the gas limit.

Reply:
Removed.